rasa/importers/multi_project.py

```diff
@@ -95,6 +95,10 @@ def _init_from_directory(self, path: Text):
                     # Check next file
                     continue
 
+                if data.is_conversation_test_file(full_path):
+                    # skip conversation test files
+                    continue
```

Comment:
instead of omitting them, can we please store them in a different list (e.g. `self._e2e_story_paths`) and return either `self._story_paths` or `self._e2e_story_paths` depending on whether `e2e` is `True` / `False` in `get_stories`?